mu-plugins/blocks/ratings-stars/index.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Block Name: Ratings (stars)
+ * Description: The avgerage rating of this theme displayed as stars.
```

Comment:
```suggestion
 * Description: The average rating of this theme displayed as stars.
```
Pretty sure this typo's my fault  😆